applications/NurbsBrepApplication/CMakeLists.txt

```diff
@@ -2,7 +2,10 @@ set(CMAKE_INCLUDE_CURRENT_DIR ON)
 
 message("**** configuring KratosNurbsBrepApplication ****")
 
-include_directories( ${CMAKE_SOURCE_DIR}/kratos )
+include_directories(
+    ${CMAKE_SOURCE_DIR}/kratos
+    ${ANURBS_ROOT}/include
```

Comment:
this works without including ANurbs as a submodule?

Reply:
At the moment it woks by specifing `ANURBS_ROOT` in your build script (the same as for Eigen). I need to wait for a final decision to integrate the code into Kratos.